machine_learning_hep/data/data_prod_20210223/database_variations_LcpK0spp_jet_FF.yml

```diff
@@ -265,7 +265,7 @@ categories:
         rms_both_sides: true
         variations:
             cutvar:
-                activate: [yes, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no]
+                activate: [yes, no, yes, yes, yes, no, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes, yes]
```

Comment:
Why are there two `no`s?

Reply:
the fits seemed unstable with these so i removed them